src/erlcloud_kinesis.erl

```diff
@@ -72,6 +78,30 @@ configure(AccessKeyID, SecretAccessKey, Host, Port) ->
 
 default_config() -> erlcloud_aws:default_config().
 
+dynamize_option({exclusive_start_shard_id, Value}) ->
```

Comment:
Need support for `{stream_name, StreamName}` here so one can build a complete implementation without having to call /3 *and* /2.